android/app/src/main/java/com/papel/data/Article.java

```diff
@@ -14,25 +14,26 @@
     private String date;
     private ArrayList<Comment> comments;
     private int voteCount;
-    private double rank;
+    private Date dateObj;
 
     public Article() {
     }
 
-    public Article(String id, String title, String body, String authorId, String authorName, int voteCount, double rank, String date) {
+    public Article(String id, String title, String body, String authorId, String authorName, int voteCount, String date) {
         this.id = id;
         this.title = title;
         this.body = body;
         this.authorId = authorId;
         this.authorName = authorName;
         this.voteCount = voteCount;
-        this.rank = rank;
         this.date = date;
 
         try {
             SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ", Locale.US);
             Date dateObj = formatter.parse(date.replaceAll("Z$", "+0000"));
-            SimpleDateFormat formatter2 = new SimpleDateFormat("dd MMM yy • HH:mm a", Locale.US);
+            this.dateObj = dateObj;
+            SimpleDateFormat formatter2 = new SimpleDateFormat("EEEE, MMM dd, yyyy HH:mm:ss a", Locale.US);
```

Comment:
To solve the overlapping names and dates, we removed the seconds from the date format in #216  Why we add the seconds again?

Reply:
Sorry, that was a mistake. Thanks for pointing it out. I fixed it.